### PR TITLE
adds search page and displays email next to comments

### LINKED ISSUE #28: Create Search.html page
Create and Style Search.html to look consistent with the rest of the website

### PR DESCRIPTION
**Pull request by TylerHernandez**
adds email to display next to comments

### CODE REVIEW

**Review by yjzhang111 — COMMENTED** [review]
Code looks good, but if we merge it to main and deploy, user emails associated with all comments will be shown, which is not exactly what we want.

**Review by dag13 — DISMISSED** [review]

**Comment by dag13** [comment]
There's some conflicts that need to be resolved before this can be merged.

File: src/main/java/com/google/sps/servlets/LoginServlet.java
Comment (mutisog) [inline src/main/java/com/google/sps/servlets/LoginServlet.java]: is this whitespace necessary?

File: src/main/webapp/coffeeshop.html
Comment (mutisog) [inline src/main/webapp/coffeeshop.html:12]: Have we tested all 3 functions are called sequentially and work as expected?
Also, as an FYI, read about alternative approaches here: https://www.htmlgoodies.com/beyond/javascript/article.php/3724571/using-multiple-javascript-onload-functions.htm

File: src/main/webapp/search.html
Comment (mutisog) [inline src/main/webapp/search.html]: Is this still MVP work?

File: src/main/webapp/search.html
Comment (mutisog) [inline src/main/webapp/search.html]: lowercase?

**Review by mutisog — COMMENTED** [review]
Left some comments.

File: src/main/webapp/coffeeshop.html
Comment (mutisog) [reply src/main/webapp/coffeeshop.html:12]: Also: https://stackoverflow.com/questions/33785313/javascript-loading-multiple-functions-onload

File: src/main/java/com/google/sps/servlets/LoginServlet.java
Comment (TylerHernandez) [reply src/main/java/com/google/sps/servlets/LoginServlet.java]: no

File: src/main/webapp/coffeeshop.html
Comment (TylerHernandez) [reply src/main/webapp/coffeeshop.html:12]: none of the three function calls are dependent on each other, so whichever way the functions are called (as long as all of them are called) will suffice for our plans

File: src/main/webapp/search.html
Comment (TylerHernandez) [reply src/main/webapp/search.html]: No, i will change the title

File: src/main/webapp/search.html
Comment (TylerHernandez) [reply src/main/webapp/search.html]: yes, i'm unsure why but I have changed email to lowercase weeks ago and this will be the third time i will do it again, but it keeps showing up as uppercase.

**Review by dag13 — DISMISSED** [review]

**Comment by TylerHernandez** [comment]
> There's some conflicts that need to be resolved before this can be merged.

I have just resolved the conflicts, could you re-review?

**Review by yjzhang111 — APPROVED** [review]

**Review by dag13 — APPROVED** [review]

File: src/main/java/com/google/sps/servlets/CreateUsername.java
Comment (mutisog) [inline src/main/java/com/google/sps/servlets/CreateUsername.java:56]: Is the System.out.println needed?

File: src/main/java/com/google/sps/servlets/CreateUsername.java
Comment (mutisog) [inline src/main/java/com/google/sps/servlets/CreateUsername.java:74]: "set" is too generic and loses info. Rename to "usernames" or similar.

File: src/main/java/com/google/sps/servlets/GetUsername.java
Comment (mutisog) [inline src/main/java/com/google/sps/servlets/GetUsername.java:44]: Remove the print statement

File: src/main/webapp/script.js
Comment (mutisog) [inline src/main/webapp/script.js:60]: typo: "fconwill"

File: src/main/webapp/script.js
Comment (mutisog) [inline src/main/webapp/script.js:58]: This todo is not clear, please explain/rephrase.

File: src/main/webapp/script.js
Comment (mutisog) [inline src/main/webapp/script.js:61]: nit: put the todo above the function

**Review by mutisog — APPROVED** [review]
Left some comments. Feel free to submit after addressing without waiting for additional approval from Albert or me.

**Pull request merged**
